code/__DEFINES/calibers.dm

```diff
@@ -6,6 +6,7 @@
 #define CALIBER_9X19 "9x19mm Parabellum"
 #define CALIBER_9X19_TRANQUILIZER "9x19mm Tranquilizer"
 #define CALIBER_45ACP ".45 ACP"
+#define CALIBER_AP_45ACP "Armor Piercing .45 ACP"
```

Comment:
```suggestion
#define CALIBER_AP_45ACP "Armor Piercing .45 ACP"
```
You know you can use the `CALIBER_45ACP ".45 ACP"`, and make them use AP bullets from there, right?